pzsd_bot/ui/triggers/modals.py

```diff
@@ -34,19 +54,44 @@ async def get_input(
                     "%s submitted trigger with invalid regex, doing nothing.",
                     interaction.user.name,
                 )
-                await interaction.respond("Invalid regex, failed to add trigger.")
+                await interaction.respond(
+                    "Invalid regex, failed to add trigger.", ephemeral=True
+                )
                 return
         else:
             patterns = self.children[0].value.lower().split(",")
 
         responses = self.children[1].value.splitlines()
 
+        if self.response_type is TriggerResponseType.reaction:
+            responses = [
+                emoji.emojize(response.strip(), language="alias")
+                for response in responses
```

Comment:
Why you looping through responses here and then again right after, just move this logic to the other loop

Reply:
Real reason is that it was late and i was too lazy to write it differently. The second loop doesn't mutate `responses`, it's not terribly convenient to just combine the logic here.

Reply:
suppose I could do this, feels kinda awkward:
```py
            for i, response in enumerate(responses):
                responses[i] = emoji.emojize(response.strip(), language="alias")

                if not self.is_valid_emoji(response):
                    logger.info(
                        "%s submitted trigger with invalid reaction, doing nothing.",
                        interaction.user.name,
                    )
                    await interaction.respond(
                        f"Invalid reaction `{response}`, failed to add trigger.",
                        ephemeral=True,
                    )
                    return
```
Was trying to come up with something more elegant and got impatient.

Reply:
I guess responses won't ever get that large, I'll accept it